src/Unosquare.RaspberryIO.Peripherals/AccelerometerGY521.cs

```diff
@@ -0,0 +1,205 @@
+﻿namespace Unosquare.RaspberryIO.Peripherals
+{
+    using System;
+    using System.Threading;
+    using Unosquare.RaspberryIO.Abstractions;
+    using Unosquare.RaspberryIO.Abstractions.Native;
+
+    /// <summary>
+    /// Enum FSSEL
+    /// </summary>
+    internal enum FSSEL
+    {
+        /// <summary>
+        /// Gyroscope Full Scale Range 250
+        /// </summary>
+        FSR250,
+
+        /// <summary>
+        /// Gyroscope Full Scale Range 500
+        /// </summary>
+        FSR500,
+
+        /// <summary>
+        /// Gyroscope Full Scale Range 1000
+        /// </summary>
+        FSR1000,
+
+        /// <summary>
+        /// Gyroscope Full Scale Range 2000
+        /// </summary>
+        FSR2000,
+    }
+
+    /// <summary>
+    /// Enum AFSSEL
+    /// </summary>
+    internal enum AFSSEL
+    {
+        /// <summary>
+        /// Accelerometer Full Scale Range 2g
+        /// </summary>
+        FSR2G,
+
+        /// <summary>
+        /// Accelerometer Full Scale Range 4g
+        /// </summary>
+        FSR4G,
+
+        /// <summary>
+        /// Accelerometer Full Scale Range 8g
+        /// </summary>
+        FSR8G,
+
+        /// <summary>
+        /// Accelerometer Full Scale Range 16g
+        /// </summary>
+        FSR16G,
+    }
+
+    /// <summary>
+    /// Implements settings for GY-521 accelerometer peripheral.
+    /// </summary>
+    public sealed class AccelerometerGY521 : IDisposable
+    {
+        private const int PwrMgmt1 = 0x6b;
+        private const int GyroSensitivity = 0x1b;
+        private const int AcclSensitivity = 0x1c;
+
+        private readonly ushort[] AFSSel = new ushort[] { 0x4000, 0x2000, 0x1000, 0x0800 };
```

Comment:
I don't know where these values came from or what does it means.

Reply:
These are the range values.